cmd/virt-handler/Dockerfile

```diff
@@ -28,4 +28,9 @@ RUN yum -y install wget && \
 
 COPY virt-handler /virt-handler
 
+RUN groupadd -g 993 libvirt
```

Comment:
We have to explicitly set the gid and uid in the libvirt image we are using. Then we can synchronize them.